src/main/java/bio/terra/common/DaoKeyHolder.java

```diff
@@ -25,6 +25,10 @@ public Instant getCreatedDate() {
         return null;
     }
 
+    public String getString(String fieldName) {
+        return getField(fieldName, String.class);
+    }
```

Comment:
Worth writing a unit test for, or small enough to piggy-back on existing tests for `getField`?